python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/asset_decorator.py

```diff
@@ -0,0 +1,181 @@
+import re
+from typing import Any, Callable, Iterable, Mapping, Optional
+
+from dagster import (
+    AssetKey,
+    AssetsDefinition,
+    AssetSpec,
+    BackfillPolicy,
+    FreshnessPolicy,
+    PartitionsDefinition,
+    multi_asset,
+)
+from dagster._annotations import public
+from dagster._core.definitions.asset_dep import CoercibleToAssetDep
+from dagster._core.definitions.auto_materialize_policy import AutoMaterializePolicy
+
+from dagster_embedded_elt.sling.sling_replication import SlingReplicationParam, validate_replication
+
+
+def get_streams_from_replication(replication: Mapping[str, Any]) -> Iterable[str]:
```

Comment:
Just to be explicit and match what Sling has in their docs.

```suggestion
def get_streams_from_replication(replication_config: Mapping[str, Any]) -> Iterable[str]:
```